concent_api/core/decorators.py

```diff
@@ -1,3 +1,4 @@
+import time
```

Comment:
Please change it into `from time import time`

Reply:
Done